src/error/specification

```diff
@@ -1,1347 +1,44 @@
-number:0
-description:a test for an exception specification
+number:00000
```

Comment:
I definitely like the shortness of this file. However, while I think I can grasp what a

- parsing,
- conflict,
- timeout, and
- validation

error is, I am not sure about the categories. Can you please add a short description and one example for the other errors? Otherwise, I do not think I will be able to add any useful comment to commit 058d194c.

Reply:
I am currently working on a description which will come in the next commit! :)